subgraphs/euler-finance/src/mappings/handlers.ts

```diff
@@ -144,8 +144,8 @@ function updateProtocolAndMarkets(block: ethereum.Block): void {
   let protocolUtility = getOrCreateProtocolUtility(blockNumber);
   let markets = protocolUtility.markets;
 
-  if (protocolUtility.lastBlockNumber >= blockNumber ) {
-    return; // Do this update every block
+  if (protocolUtility.lastBlockNumber >= blockNumber - 120 ) {
+    return; // Do this update every 120 blocks
```

Comment:
how come every 120 blocks? maybe include some reasoning/methodology so it doesn't seem like its an arbitrary number pulled out of nowhere